### ISSUE TITLE
Pulseq Integration ? 

### ISSUE THREAD

**Issue by paquiteau** [mind-inria/mri-nufft#102]
At ISMRM 2024, peoples asks if there were any Pulseq integration. 

I am not familiar with Pulseq, but let's have this here for future reference and discussion.

**Comment by chaithyagr** [comment]
I dont think this makes sense for mri-nufft, they mentioned mostly for running arbitrary gradients. This is a separate repository  (more on sequence development) and not at all related to this repo which is mostly on trajectories and NUFFT. I think we can close this

**Comment by paquiteau** [comment]
My point would be that if Pulseq has some support for arbitrary gradient files in a specific format, we could support it (just like we support to export to nsp `.bin` format)
